pkg/output/output.go

```diff
@@ -210,6 +216,28 @@ func (b *Handler) writeSIGRetestBadge(name, filePath string, data types.RunningA
 	return badge.Render(name, badgeString, color, f)
 }
 
+func (b *Handler) writeJobFailureBadges(data types.RunningAverageDataItem, levels *Levels) error {
+	failedJobLeaders := data.FailedJobLeaderBoard
+	basePath, err := b.initializeSourcePath()
+	if err != nil {
+		return err
+	}
+	for i, job := range failedJobLeaders[:10] {
+		filePath := filepath.Join(basePath, fmt.Sprintf("failedjob%s.svg", strconv.Itoa(i+1)))
+		color := BadgeColor(float64(job.FailureCount), levels)
+		f, err := os.Create(filePath)
+		if err != nil {
+			return err
+		}
+		defer f.Close()
+
+		badgeString := fmt.Sprintf("%.0f", float64(job.FailureCount))
```

Comment:
come to think of it, can you print here FailureCount/AttemptCount? This would provide us the metric I care about: how often does the lane fail on mergeable code.